src/copilot/app/rag_api.py

```diff
@@ -30,23 +39,26 @@
 )
 
 
-@app.post("/rag/process", summary="Process RAG query endpoint", response_description="Return result from processing RAG query", status_code=200)
+@app.post("/query",
+          summary="Process RAG query endpoint",
+          response_description="Return result from processing RAG query",
+          status_code=200)
 async def process_query(request: RAGRequest):
     content = await processor.process(request)
     return StreamingResponse(content, media_type="text/event-stream")
 
 
-@app.post("/rag/docs", summary="Retrieve context docs endpoint", response_description="Return context docs from semantic search", status_code=200)
+@app.post("/context",
```

Comment:
I would stick with "/docs" as the route for retrieving docs although we don't use the endpoint at the moment. It's considered the standard nomenclature (to be more precise "context_docs", but I think /docs is sufficient).

Reply:
I changed to /context_docs. I want to avoid confusion with the swagger doc